README.md

```diff
@@ -88,23 +89,20 @@ This example showcases that it's possible to have a situation in which the cooki
 
    * “Schemeful Same-Site” would also allow this cookie to be sent as we're still in a same-site context when considering scheme: the user is on http://website.example and the cookie wants to be sent to http://website.example
 
-   * “Scheme-Bound Cookies” would not allow this cookie to be sent as the connection isn't the same as the one the cookie was set by: the cookie was set on https and is trying to be sent on http.
+   * “Scheme-Bound Cookies” would not allow this cookie to be sent as the scheme isn't the same as the one the cookie was set by: the cookie was set by https and is trying to be sent to http.
 
-4. Immediately after the browser checks to see if it can send the cookie to https://website.example/readsacookie2.jpg
+4. Next the browser checks to see if it can send the cookie to https://website.example/readsacookie2.jpg
```

Comment:
Might be clearer to:
Next, the browser makes a request to https://website.example/readsacookie2.jpg and checks to see if it can send the cookie.

Reply:
Reworked the wording a bit, I think it addresses your concern.